### DISCUSSION TITLE
Show the latest searches and the most frequent searches first

### DISCUSSION THREAD

**Discussion by devistar** [Flow-Launcher/Flow.Launcher#1062] [Ideas]
Show the latest searches and the most frequent searches first.

For example, if in the last search I typed "Control pannel", when I type "C" show me "Control pannel" for the first result.

If I have searched for "Vivaldi" several times, if I type "V" display "Vivaldi" first.

**Comment by jjw24** [comment]
It's actually designed to do that but you will need to select for example 'control panel' with 'c'  search word 3 or a bit more times and then it will show up at the top.

If you always have only one result for your letter, you can go to the context menu of the desired result and select 'set as topmost in this query'.
